pkg/daemon/certificate_writer.go

```diff
@@ -221,7 +221,7 @@ func (dn *Daemon) syncControllerConfigHandler(key string) error {
 							}
 
 							pathToData[kubeConfigPath] = newData
-							klog.Infof("Writing new Data to /etc/kubernetes/kubeconfig: %s", string(newData))
+							klog.Infof("Writing new Data to /etc/kubernetes/kubeconfig")
```

Comment:
nit: technically this should be the unformatted `info` but it shouldn't affect anything